src/GitHub/Data/PullRequests.hs

```diff
@@ -191,7 +193,7 @@ instance FromJSON SimplePullRequest where
         <*> o .: "html_url"
         <*> o .: "updated_at"
         <*> o .:? "body"
-        <*> o .: "assignees"
```

Comment:
Check: https://api.github.com/repos/phadej/github/pulls/285, there are both keys "assignee" and "assignees"